Actions/ReadSecrets/ReadSecretsHelper.psm1

```diff
@@ -84,6 +84,10 @@ function GetGithubSecret {
 }
 
 function GetKeyVaultCredentials {
+    Param(
+        [string] $testSecretName = $null
```

Comment:
It looks like we always want to test if the provided credentials are enough to read secrets. So, a parameter isn't needed.